### PR TITLE
Enhance User Feedback on 'Submit Resource' Button

### LINKED ISSUE #237: [Bug] Enhance User Feedback on 'Submit Resource' Button
## Description
When navigating to the "Submit Resource" page, I observed that there's no hover feedback on the button itself. Unlike other buttons where we typically get a visual indication upon hovering, this one seems to lack that interaction.

While it's a small detail, I believe incorporating hover feedback on this button would enhance user experience and ensure consistency across the platform.

## Reproduction

1. Go to resource archive website
2.  Click on Submit Resource button
3. See error.

## Expectation
 Anticipating swift attention to add hover feedback on the 'Submit Resource' button for improved user interaction.

### PR DESCRIPTION
**Pull request by piyushs131**
## Fixes

- Fixes #237 by @piyushs131

## Description
When navigating to the "Submit Resource" page, I observed that there's no hover feedback on the button itself. Unlike other buttons where we typically get a visual indication upon hovering, this one seems to lack that interaction.

While it's a small detail, I believe incorporating hover feedback on this button would enhance user experience and ensure consistency across the platform.
## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Xaid-vfx** [comment]
#237 is not yet approved, wait for it to be approved by mentors.

File: docs/Gemfile.lock [hunk 1]
```diff
@@ -270,6 +270,7 @@ GEM
 PLATFORMS
   aarch64-linux
   arm64-darwin-23
+  x86_64-linux
```
Comment (Murdock9803) [inline docs/Gemfile.lock:273]: This additional line in `Gemfile.lock` file emerges at the time of local development. It is of no use here so remove 👍

**Comment by ansh258** [comment]
Thanks for your feedback, I will make sure to focus on keeping the improvements to the button straightforward and limit changes to CSS additions only. Thanks for guiding me

**Comment by piyushs131** [comment]
@possumbilities Thanks for your feedback. I will make sure to focus on keeping the improvements to the button straightforward and limit changes to CSS additions only. Thanks for guiding me

**Pull request closed without merging**